portal/server/routes/contract/bond/index.js

```diff
@@ -220,8 +220,8 @@ router.post('/contract/:_id/bond/:bondId/financial-details', async (req, res) =>
 });
 
 router.post('/contract/:_id/bond/:bondId/financial-details/save-go-back', provide([BOND]), async (req, res) => {
-  const bondPayload = filterBondFinancialDetailsPayload(req.body);
-  return saveFacilityAndGoBackToDeal(req, res, bondPayload);
+  const sanitizedPayload = filterBondFinancialDetailsPayload(req.body, bondFinancialDetailsPayloadProperties, ['currency']);
```

Comment:
Sonarcloud found real issue here.
Function `filterBondFinancialDetailsPayload` expects just one parameter, no need to have change function call at line 223.

Reply:
Changed to `const sanitizedPayload = filterBondFinancialDetailsPayload(req.body);`